src/wg21.cls

```diff
@@ -465,8 +465,8 @@
 \newcommand{\requires}{\Fundesc{Requires}}
 \newcommand{\mandates}{\Fundesc{Mandates}}
 \newcommand{\constraints}{\Fundesc{Constraints}}
-\newcommand{\expects}{\Fundesc{Expects}}
-\newcommand{\ensures}{\Fundesc{Ensures}}
+\newcommand{\expects}{\Fundesc{Preconditions}}
+\newcommand{\ensures}{\Fundesc{Postconditions}}
```

Comment:
That's certainly one way to fix it :)

Reply:
This is how they fixed them in the IS, too =)